### DISCUSSION TITLE
Can this be used to convert a Tree into svg ?

### DISCUSSION THREAD

**Discussion by baseplate-admin** [DioxusLabs/taffy#652] [Q&A] [answered]
Hi,

Can i use `taffy` to convert tree to svg?

The plan i have in mind is:

1. parse html+css
2. make a tree out of that
3. Convert tree into svg
4. use resvg to get a png.

If someone can guide me to some resources that might help me in my journey, it'll be greatly appreciated :D

**Comment by nicoburns** [comment, marked as answer]
Is your ultimate aim here to render HTML+CSS into a png? If so then you might want to look at https://github.com/jkelleyrtp/stylo-dioxus (which uses Taffy). It's not quite ready yet (lots of rendering bugs need to be fixed) and it currently renders to screen rather than to a png. But it will be ideal for that purpose once it's more mature.

**Reply by baseplate-admin** [reply]
Thanks for pointing me towards `stylo-dioxus` my plan in mind was something similar
